### PR TITLE
OCLOMRS-978: Update SignUp links to new OCL links

### PR DESCRIPTION
**Pull request by suruchee**
#OCLOMRS-978:
[Update SignUp links to new OCL links](https://issues.openmrs.org/browse/OCLOMRS-978)

### CODE REVIEW

File: src/utils/constants.ts [hunk 1]
```diff
@@ -8,7 +8,7 @@ export const TRADITIONAL_OCL_URL =
   window.TRADITIONAL_OCL_HOST || "https://qa.openconceptlab.org";
 export const OCL_SIGNUP_URL =
   // @ts-ignore OCL_SIGNUP_URL is injected at runtime via index.html
-  window.OCL_SIGNUP_URL || "https://qa.openconceptlab.org/accounts/signup/";
+  window.OCL_SIGNUP_URL || "https://qa.openconceptlab.org/#/accounts/signup/";
```
Comment (HerbertYiga) [inline src/utils/constants.ts:11]: i find the changed link working perfect cc @hadijahkyampeire

Comment (hadijahkyampeire) [reply src/utils/constants.ts:11]: Thanks @HerbertYiga for testing this I will test it and approve

**Review by hadijahkyampeire — COMMENTED** [review]
@suruchee, I have two issues with this, 
1. The link we are using takes one to the whole site why don't we use the one the links to the signup form directly? i.e `https://app.openconceptlab.org/#/accounts/signup`.
2. You need to make changes to the docker-compose file in OCL because I tried to run using docker and the signup link was broken. So the URL we decide to use should be changed in two places, constants and docker-compose file line 11.

**Comment by hadijahkyampeire** [comment]
Also, rebase with master to get the fix for those tests.

**Comment by suruchee** [comment]
@hadijahkyampeire thanks for your review, I actually updated it as it was before. please review the updated link.

**Review by hadijahkyampeire — APPROVED** [review]
LGTM

File: docker-compose.yml [hunk 2]
```diff
@@ -8,7 +8,7 @@ services:
     environment:
     - OCL_API_HOST=${OCL_API_HOST:-https://api.qa.openconceptlab.org/}
     - TRADITIONAL_OCL_HOST=${TRADITIONAL_OCL_HOST:-https://qa.openconceptlab.org}
-    - OCL_SIGNUP_URL=${OCL_SIGNUP_URL:-https://qa.openconceptlab.org/accounts/signup/}
+    - OCL_SIGNUP_URL=${OCL_SIGNUP_URL:-https://app.openconceptlab.org/#/accounts/signup}
```
Comment (ibacher) [inline docker-compose.yml:11]:
```suggestion
    - OCL_SIGNUP_URL=${OCL_SIGNUP_URL:-https://app.qa.openconceptlab.org/#/accounts/signup}
```

File: src/utils/constants.ts [hunk 3]
```diff
@@ -8,7 +8,7 @@ export const TRADITIONAL_OCL_URL =
   window.TRADITIONAL_OCL_HOST || "https://qa.openconceptlab.org";
 export const OCL_SIGNUP_URL =
   // @ts-ignore OCL_SIGNUP_URL is injected at runtime via index.html
-  window.OCL_SIGNUP_URL || "https://qa.openconceptlab.org/accounts/signup/";
+  window.OCL_SIGNUP_URL || "https://app.openconceptlab.org/#/accounts/signup";
```
Comment (ibacher) [inline src/utils/constants.ts:11]:
```suggestion
  window.OCL_SIGNUP_URL || "https://app.qa.openconceptlab.org/#/accounts/signup";
```

**Review by ibacher — APPROVED** [review]

**Pull request merged**